src/lib.rs

```diff
@@ -3,7 +3,6 @@
 #![feature(exclusive_range_pattern)]
 #![feature(once_cell)]
 #![feature(c_variadic)]
-#![deny(warnings)]
```

Comment:
This caused normal builds to fail if they were warnings lol, obnoxious. I put this in the clippy check in the Github Action, that way we just get stopped on PR build rather than annoy dev process.